### ISSUE TITLE
Landscape: Publish application Internal Server Error

### ISSUE THREAD

**Issue by charlieroth** [urbit/urbit#1942]
I ran the command `:publish &publish-action [%subscribe ~rapfyr-diglyt %hoon-school-101]` to subscribe to a notebook for hoon school and received a response of `>=` so all good there. A few minutes later I get a notification on the Publish tile and when I clicked I get this screen:

![Screen Shot 2019-11-12 at 7 55 26 PM](https://user-images.githubusercontent.com/13981427/68727739-41c54480-0593-11ea-9257-07d74879e4d8.png)

I tried restarting my ship and still the same thing. I also ran `:azimuth-tracker|listen ~ %app %azimuth-tracker` to see if that would help but no difference.

**Comment by vvisigoth** [comment]
@ixv can you take a look at this?

**Comment by charlieroth** [comment]
The publish app is working for me now

**Comment by jtobin** [comment]
I'll close this for the time being as it appears to have been resolved.

**Comment by mocknotes** [comment]
In Publish I unsubscribed and resubscribed to all Notebooks to get Recent working https://github.com/urbit/urbit/issues/2079. I'm now getting "Internal Server Error There was an error while handling the request for "/~publish"." I tried refreshing Landscape and also tried restarting my ship without success. I can still use Landscape for chatting.
![Screen Shot 2019-12-11 at 3 37 39 AM](https://user-images.githubusercontent.com/822317/70619702-6d574100-1bca-11ea-93b0-4a61d41c029d.png)
